config/jobs/kubernetes-sigs/kubetest2/kubetest2-gke-presubmits.yaml

```diff
@@ -0,0 +1,42 @@
+presubmits:
+  kubernetes-sigs/kubetest2:
+  - name: gke-build-up-down-sc-make
+    decorate: true
+    run_if_changed: "kubetest2-gke/.*"
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210715-521d667-master
+        command:
+        - "runner.sh"
+        args:
+        - "./kubetest2-gke/ci-tests/buildupdown.sh"
+        - "--cluster-topology"
+        - "sc"
+        - "--build-strategy"
+        - "make"
+        securityContext:
+          privileged: true
+  - name: gke-build-up-down-mc-gkemake
+    decorate: true
+    run_if_changed: "kubetest2-gke/.*"
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210715-521d667-master
+        command:
+        - "runner.sh"
+        args:
+        - "./kubetest2-gke/ci-tests/buildupdown.sh"
+        - "--cluster-topology"
+        - "mc"
+        - "--build-strategy"
+        - "gke_make"
```

Comment:
let's drop --build in upstream jobs since we won't have the sources.

Reply:
Got it, changed to `bazel`, if it is supported for OSS.

Reply:
bazel is definitely not supported. I think it's fine to just drop build altogether and just have up down

Reply:
Thanks for the clarification. Removed.